rust/src/cli.rs

```diff
@@ -17,11 +20,14 @@ pub async fn init_vault_stack(
     stack_name: Option<String>,
     region: Option<String>,
     bucket: Option<String>,
+    quiet: bool,
 ) -> Result<()> {
     match Vault::init(stack_name, region, bucket).await? {
         CreateStackResult::Exists { data } => {
-            println!("Vault stack already initialized");
-            println!("{data}");
+            if !quiet {
+                println!("{}", "Vault stack already initialized:".bold());
+                println!("{data}");
+            }
```

Comment:
Yeah, not sure what would be nicest way to handle this. I basically wanted to have an option to control the verbosity of the output and suppress the output in particular cases like in the `exists` so it can be used similarly to for example `grep -q` which will just indicate with the exit codes if something was found or not...

Reply:
Hmm, I wonder if we should change lookup to return exit codes based if the entry was found or not and rely on that🤔. Also sending printing specific stuff to stderr instead of stdout would solve this problem as well?

Reply:
Outputting to stderr does not really help in this case since you would then need to suppress some output anyway. The anyhow error print goes there already anyway.